src/Landing/containers/Landing.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import RecommendedArticles from '../components/RecommendArticles';
+import LatestArticles from '../components/LatestArticles';
+import fetchLatestArticles from '../redux/actions/latestArticlesAction';
+
+class Landing extends Component {
+  componentWillMount() {
+    this.props.fetchLatestArticles();
```

Comment:
Try and add this to your tests as well...

Reply:
this has been worked on. thanks.